manual/guides/writing/checklist.md

```diff
@@ -51,6 +51,14 @@ This [writing checklist](/manual/guides/writing/checklist) is mostly from advice
 - [ ] A phrase of the form <verb>ion of <noun> is probably clearer as <noun> <verb>ion. (For example, convert "calculation of velocity" to "velocity calculation".)
 - [ ] Cite all software. Review [the principles](https://www.force11.org/software-citation-principles) and try [CiteAs](https://citeas.org/about) if necessary.
 - [ ] Refer to software consistently by name.
+- [ ] Each sentence/paragraph should logically follow the previous sentence/paragraph.
+- [ ] Be concise and direct.
+- [ ] Cite your sources. In particular, cite all images, methods, software, and empirical data.
```

Comment:
Line 52 mentions citing software as well. If citation errors are common enough, you could make a separate section about them to provide similar resources as in 52. Or you could combine both and make each word (e.g., software, images, methods, and empirical data) a link to an appropriate site.

Reply:
Thanks for the input! I didn't realize I was just duplicating the item